janitor/functions/utils.py

```diff
@@ -622,6 +622,27 @@ def get_index_labels(
     return index[_select_index(arg, df, axis)]
 
 
+def get_columns(group, label):
+    """
+    Helper function for selecting columns on a grouped object,
+    using the
+    [`select_columns`][janitor.functions.select.select_columns] syntax.
+
+    !!! info "New in version 0.25.0"
+
+    Args:
+        group: A Pandas GroupBy object.
+        label: column(s) to select.
+
+    Returns:
+        A pandas groupby object.
+    """
+    assert isinstance(group, (DataFrameGroupBy, SeriesGroupBy))
```

Comment:
I think using `asserts` isn't idiomatic. Could we change it to the following, please?

```suggestion
    if not isinstance(group, (DataFrameGroupBy, SeriesGroupBy)):
        raise TypeError(...)
```